src/index.html

```diff
@@ -15,5 +15,54 @@
     <h1 class="center">Hello, world! It works!</h1>
     <h2 class="center">Webpack 4 boilerplate with TypeScript is running!</h2>
     <script src="<%= htmlWebpackPlugin.files.chunks.main.entry %>"></script>
+    <script>
+
```

Comment:
Put all the logic related to the game into the `chooseNumber` function and call it